### ISSUE TITLE
Current sublicense grant is likely a loophole

### ISSUE THREAD

**Issue by ryan-lane** [ContributorCovenant/hippocratic-license#42]
With the current version of the license, I can fork a project under the hippocratic license, and release future versions with a different license. The current license requires copies of the code to include the license, but once a new license is applied, that requirement would no longer apply. This includes the ability to close source the software.

The sublicense grant is conditional upon compliance of ethical use, but doesn't require that sublicenses require ethical use.

**Comment by CoralineAda** [comment]
> With the current version of the license, I can fork a project under the hippocratic license, and release future versions with a different license.

"The above copyright notice and this License or a subsequent version published on the Hippocratic License Website (https://firstdonoharm.dev/) shall be included in all copies or substantial portions of the Software. Licensee has the option of following the terms and conditions either of the above numbered version of this License or of any subsequent version published on the Hippocratic License Website.”

I don’t see how you’re interpreting it that way?

**Comment by ryan-lane** [comment]
Sentry recently changed their license from BSD3 to BSL. Any version after that point is BSL and anything prior to that is BSD3. Does the sublicensing require that sublicenses also require ethical use? The license says that relicensing is granted providing ethical use, but doesn't explicitly say that the sublicenses themselves must require ethical use.

My concern is that the language is ambiguous in a way that would allow sublicenses that don't require the same level of restriction.

**Comment by CoralineAda** [comment]
I’ll pass this along to the legal team and see what they think. Thanks for raising this!

**Comment by ryan-lane** [comment]
You're welcome!

**Comment by ryan-lane** [comment]
I have the same general concern about modifications of the software. The license is permissive, and it reads like the act of modifying must meet the license, but that the modifications do not.

With an MIT license, for example, my modifications aren't under the same license.

**Comment by decentral1se** [comment]
Ah, excellent, thanks for raising @ryan-lane. I was also wondering about this and I think a number of other issues which were addressing other wider issues also had this point contained within them. For example, *I think* people were referring to this sub-licensing issue when talking about "strong copyleft". That's a rough guess though. Refs: https://github.com/EthicalSource/hippocratic-license/issues/31#issue-527604965 and https://github.com/EthicalSource/hippocratic-license/issues/21#issuecomment-547557302.

**Comment by paulyc** [comment]
Well i think the distinction that needs to be made here if I'm reading this right is the difference between what the copyright holder of a purely original work can do and what the copyright holder of a derivative work of something licensed under certain terms can do.

As the copyright holder of an original work they can license or relicense that work under whatever terms they want at any time they want, but they cannot revoke a license already granted assuming it was granted in perpetuity which I've noticed many licenses do not seem to specify any duration at all. It sounds like thats what happened in the sentry case but I'm not sure i don't know all the details.

However, the author of a derivative work of another solely licensed under one specific set of terms cannot change any of those terms for the derivative work. That is just a feature of the law unless the license specifically allowed the work to be relicensed under any terms and then there would be no point having any terms at all so i don't think that's what the license currently says.

 though, that is a risk of allowing licensing under "any subsequent version" of a license since, who knows what any subsequent version might say, and who has the authority to create any subsequent version so it would be up to a judge to determine if a "subsequent version" was in fact a subsequent version of the same license, kind of like the argument over in cryptocracklandia over what is the legitimate version of Bitcoin since the original author has chosen to remain silent on the subject!

**Comment by decentral1se** [comment]
> Well i think the distinction that needs to be made here if I'm reading this right is the difference between what the copyright holder of a purely original work can do and what the copyright holder of a derivative work of something licensed under certain terms can do.

Yep. From what I can tell, that comes under the idea of "permissive/non-permissive" licenses which described by https://en.wikipedia.org/wiki/Permissive_software_license can be summarised as "Permissive licenses do not try to guarantee that future versions of the software will remain free and publicly available, in contrast to copyleft licenses, which have reciprocity requirements which try to enforce this". Which as I understand, is the what the original poster here is getting at when saying: "With the current version of the license, I can fork a project under the hippocratic license, and release future versions with a different license. "

**Comment by nateberkopec** [comment]
This loophole is closed in 2.0+ (sort of):

> The above copyright notice and this License or a subsequent version published on the Hippocratic License Website (https://firstdonoharm.dev/) shall be included in all copies or substantial portions of the Software. Licensee has the option of following the terms and conditions either of the above numbered version of this License or of any subsequent version published on the Hippocratic License Website

So, the only group that can take advantage of the "loophole" is now whoever controls firstdonoharm.dev. No one else can fork and create a more permissive license.

**Comment by decentral1se** [comment]
Ok interesting! This is of course, problematic to have this power outside of the rights holders on the specific project being used, but this is good information to know nonetheless!
